doc/security.md

```diff
@@ -407,7 +407,7 @@ learning about our users' activities (and thus maintaining user privacy):
   but we intentionally do not do that, because that would expose to an
   external unrelated site what our users are doing without their knowledge.
   We instead use the approach described in
-  ["Responsible Social Share Links" by Jonathan Suh (March 26, 2015), specifically using share URLs](https://jonsuh.com/blog/social-share-links/#use-share-urls).
+  ["Responsible Social Share Links" by Jonathan Such (March 26, 2015), specifically using share URLs](https://jonsuh.com/blog/social-share-links/#use-share-urls).
```

Comment:
Not a spelling error

Reply:
Sorry I didn't catch that one -- I added `Suh` to the ignores for codespell, reran, force pushed results. Should not happen again.